ext/bcmath/bcmath.c

```diff
@@ -152,7 +152,11 @@ PHP_FUNCTION(bcadd)
 	ZEND_PARSE_PARAMETERS_END();
 
 	if (ZEND_NUM_ARGS() == 3) {
-		scale = (int) (scale_param < 0 ? 0 : scale_param);
+	    if (scale_param < 0 || scale_param > INT_MAX) {
+	        zend_argument_value_error(3, "must be greater than or equal to 0 and less than INT_MAX (%d)", INT_MAX);
```

Comment:
you could say `must be between 0 and %d`, where `%d` is `INT_MAX`. In other places, the two cases are separated (to `>= 0` and `< INT_MAX`), and we just say "is too big" in the latter case.

P.S: it's a nice improvement, thanks for fixing this issue :)

Reply:
done